src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
+
+
     public String getRandomColor() {
-        return null;
+
+        int colors = new Random().nextInt(Colors.values().length);
```

Comment:
Creating a new instance of Random every time the 'getRandomColor' method is called is not efficient. Instead, you should create a single Random instance as a class field and reuse it.